.mergify.yml

```diff
@@ -0,0 +1,18 @@
+# See https://doc.mergify.io
+
+pull_request_rules:
+- name: auto_merge
+  conditions: 
+    - -title~=(WIP|wip)
+    - -merged
+    - -closed
+    - -locked
+    - -label=blocked
+    - "#approved-reviews-by>=1"
+    - "#changes-requested-reviews-by=0"
+    - status-success=continuous-integration/travis-ci/pr
+    - status-success=AWS CodeBuild us-east-1 (aws-cdk-build)
+  actions:
+    merge:
```

Comment:
We need this to also automatically update the branch from master before the merge (ideally this should happen immediately when the branch loses sync with master, even if the PR is not ready to be merged to master yet in order to provide visibility and quick feedback.

Reply:
That is part of the `strict: smart` parameter of the merge. Moving forward we can add another rule to regularly sync out-of-date files rather than sync them after approval, but I want to get something online